modules/apps/diff/diff-impl/build.gradle

```diff
@@ -0,0 +1,12 @@
+dependencies {
+	compileInclude group: "com.liferay", name: "org.outerj.daisy.diff", version: "1.2.0-atlassian-hosted.LIFERAY-PATCHED-1"
+	compileInclude group: "net.sourceforge.nekohtml", name: "nekohtml", version: "1.9.22"
+	compileInclude group: "org.eclipse.core", name: "runtime", version: "20070801"
```

Comment:
@guilhermedcamacho shouldn't this ```compileInclude group: "org.eclipse.core", name: "runtime", version: "20070801"``` be removed from lib/portal? Any reason to include it here and still keep the lib/portal one?

Reply:
No reason to keep it, I've just removed and force pushed, thanks!